fontforgeexe/fontview.c

```diff
@@ -7047,7 +7055,17 @@ return;
 
     fontview_ready = true;
 
+// The tools menu handles its own translation. I would rather do this by testing
+// whether ti.text_untranslated is already null but there are hundreds of missing
+// initializers for that field in the menu layout code.
+#ifndef _NO_PYTHON
+    GMenuItem2 *t = FVGetToolsSubmenu();
+    FVSetToolsSubmenu(NULL);
+#endif
```

Comment:
What side effect are we trying to bypass here?

Reply:
Ah -- The localization of the python sub-menu is handled differently and we don't want `mb2DoGetText()` to write over it for us.

I wanted to handle this by pre-setting `.ti.text_untranslated` to non-NULL and modifying `mb2DoGetText()` to only set the fields if `text_untranslated` is NULL. I believe the reason I couldn't do that is because all of the static `gtextinfo` initializers, including those in all the menu entries, poop out before the `text_untranslated` field (which was added later). Therefore there's often garbage in the field and you can't use its initial value for anything. \*sigh\*

Reply:
(maybe the menus aren't a problem because they're static, but I know I ran into it in some cases.)